AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin+DataStoreBaseBehavior.swift

```diff
@@ -106,6 +106,24 @@ extension AWSDataStorePlugin: DataStoreBaseBehavior {
                completion: publishingCompletion)
     }
 
+    public func delete<M: Model>(_ modelType: M.Type,
+                                 where predicate: @escaping () -> QueryPredicate,
```

Comment:
can replace `() -> QueryPredicate` with the QueryPredicateFactory typealias

Reply:
Thanks, i'll update this so that when we make a decision to remove QueryPredicateFactory, it'll be easier to find!